python/pyspark/sql/dataframe.py

```diff
@@ -1881,6 +1881,28 @@ def toDF(self, *cols):
         jdf = self._jdf.toDF(self._jseq(cols))
         return DataFrame(jdf, self.sql_ctx)
 
+    @since(2.4)
```

Comment:
Could we put this API between `def columns(self):` and `def alias(self, alias):`?